spring-cloud-contract-tools/spring-cloud-contract-spec-pact/src/main/groovy/org/springframework/cloud/contract/verifier/spec/pact/RequestResponsePactCreator.groovy

```diff
@@ -0,0 +1,166 @@
+package org.springframework.cloud.contract.verifier.spec.pact
+
+import au.com.dius.pact.consumer.ConsumerPactBuilder
+import au.com.dius.pact.consumer.dsl.DslPart
+import au.com.dius.pact.consumer.dsl.PactDslRequestWithPath
+import au.com.dius.pact.consumer.dsl.PactDslResponse
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider
+import au.com.dius.pact.model.RequestResponsePact
+import groovy.transform.CompileStatic
+import groovy.transform.PackageScope
+import org.springframework.cloud.contract.spec.Contract
+import org.springframework.cloud.contract.spec.internal.Body
+import org.springframework.cloud.contract.spec.internal.DslProperty
+import org.springframework.cloud.contract.spec.internal.ExecutionProperty
+import org.springframework.cloud.contract.spec.internal.Headers
+import org.springframework.cloud.contract.spec.internal.QueryParameters
+import org.springframework.cloud.contract.spec.internal.Request
+import org.springframework.cloud.contract.spec.internal.Response
+
+/**
+ * Creator of {@link RequestResponsePact} instances
+ *
+ * @author Tim Ysewyn
+ * @since 2.0.0
+ */
+@CompileStatic
+@PackageScope
+class RequestResponsePactCreator {
+
+	static final Closure requestDslPropertyValueExtractor = { DslProperty property -> property.serverValue }
```

Comment:
ha! you have to add a modifier e.g. `protected` if you want this to be a constant field. Otherwise if you check out the MOP, you'll see that a `private static final` field was created with a `public static getter` method :)

Reply:
Good catch! Both should be private indeed!